packages/web-components/src/components/ic-tab-context/ic-tab-context.tsx

```diff
@@ -54,10 +54,15 @@ export class TabContext {
   }
 
   /**
-   * Emitted when a user selects a tab.
+   * @deprecated This event should not be used anymore. Use icTabSelect instead.
    */
   @Event() tabSelect: EventEmitter<IcTabSelectEventDetail>;
 
+  /**
+ * Emitted when a user selects a tab.
+ */
+  @Event() icTabSelect: EventEmitter<IcTabSelectEventDetail>;
```

Comment:
Similar to my other comment - 'tabSelect' / 'onTabSelect' is still being used in the React tabs stories, I think we should change it to 'onIcTabSelect' there now